modules/installation-supported-aws-machine-types.adoc

```diff
@@ -407,8 +412,26 @@ The following Amazon Web Services (AWS) instance types are supported with {produ
 
 |===
 ====
+endif::[]
+
+ifdef::localzone[]
+Each Amazon Web Services (AWS) Local Zone supports specific instance types for an AWS service. 
+
+[NOTE]
+====
+Red Hat did not fully test the instance types included in the following chart for your AWS instance on {product-title}. 
+====
+
+.Machine types based on x86_64 architecture for AWS Local Zones
+[%collapsible]
+====
+* c5d.*
+* m5.*
+* t3.*
+====
```

Comment:
Need to be aligned with the code:
https://github.com/openshift/installer/pull/6371/files#diff-7abecca62150eb15cb02e930cd045f4d6782f3f2b3a933d3cb67c89bc6369026R64-R70

```suggestion
====
* m6i.*
* m5.*
* r5.*
* c5.*
* c5d.*
* t3.*
====
```

The order matter on the installer code, as it is the order that it will choose according to the availability. But for docs we could keep alpha order, as the original. WDYT? + @yunjiang29 

Note: the installer will not select t3, but the user is free to use as day2

Reply:
Thanks, @mtulio . I changed the list and I added a note to describe the `t3` instance type. Please let me know WDYT.